org.kde.gwenview.json

```diff
@@ -13,7 +13,12 @@
         "--socket=cups",
         "--socket=fallback-x11",
         "--socket=pulseaudio",
-        "--socket=wayland"
+        "--socket=wayland",
+        "--system-talk-name=org.freedesktop.UDisks2",
+        "--talk-name=org.freedesktop.ScreenSaver",
```

Comment:
is this relevant when kde supports [Inihibt](https://github.com/KDE/xdg-desktop-portal-kde/commit/9031a18505e3ecdceede99542222cad90fe15fb8) portal?